### ISSUE TITLE
wrong results on emorynlp

### ISSUE THREAD

**Issue by fpcsong** [something678/TodKat#13]
Thanks for your nice work for erc, in the file ```src/DialogEvaluator_emory.py``` line 145, macro f1 only contains 6 labels, without the ```neutral ```

```
        macro_f1 = metrics.f1_score(
            pred_list,
            grd_list,
            labels=[1, 2, 3, 4, 5, 6],
            average='weighted')
        logging.info("Weighted F1-macro with neutral: {:.4f} ({}/{})\n".format(macro_f1, correct, total))
```
labels here should be [0, 1, 2, 3, 4, 5, 6]

**Comment by fpcsong** [comment]
Also, metrics.f1_score was used in a wrong way, see [doc](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)

**Comment by something678** [comment]
Hi, thanks for finding this error. I have trained a new model and updated the results, please check.
